core/translation_service.py

```diff
@@ -0,0 +1,101 @@
+            return hjson.load(f)
+
+    # This method will load another language, defined in the param 'lang'
+    def reload_translation(self, lang):
+        self.bot.send_private_channel_message("My language got changed. reloading translations...",
+                                              fire_outgoing_event=False)
+        self.bot.send_org_message("My language got changed. reloading translations...",
+                                  fire_outgoing_event=False)
+        self.event_service.fire_event("reload_translation")
+        self.language = lang
+        self.setting_service.set_value("language", lang)
+        for k1 in self.strings:
+            for callback in self.translation_callbacks.get(k1):
+                self.update_msg(k1, callback)
+
+    #updates the msg's
+    def update_msg(self, category, callback):
+        data = callback()
+        for k in data:
+            if self.strings.get(category):
+                self.strings[category][k] = data[k].get(self.language) or data[k].get("en_US")
+            else:
+                self.strings[category] = {k: data[k].get(self.language) or data[k].get("en_US")}
+
+    #
+    # the param 'variables' accepts dictionary's ONLY.
+    #
+    def get_response(self, category, key, variables=None):
+        if variables is None:
+            variables = {}
+        msg = ""
+        try:
+            if self.strings.get(category):
+                if self.strings.get(category).get(key):
+                    if isinstance(self.strings.get(category).get(key), list):
+                        for line in self.strings.get(category).get(key):
+                            msg += line.format(**variables)
+                    else:
+                        msg = self.strings.get(category).get(key).format(**variables)
+                else:
+                    self.logger.error("translating key '{key}' in category '{category}' not found"
+                                      .format(key=key, category=category))
+                    msg = "<red>Error: message for translation key <orange>'{key}'<end> not found.<end>" \
+                        .format(key=key)
+            else:
+                self.logger.error("translation category '{category}' was not found".format(category=category))
+
+                msg = "<red>Error: translating category <orange>'{category}'<end> not found.<end>" \
+                    .format(category=category)
+        except KeyError as e:
+            self.logger.error(
+                "translating error category: {mod} key: {key} with params: {params}\n Error: {stcktr}"
+                .format(mod=category, key=key, params=variables, stcktr=e))
+            msg = "<red>Error: translating error category: <orange>{mod}<end> key: <orange>{key}<end>" \
+                  " with params: <orange>{params}<end>".format(mod=category, key=key, params=variables)
+        finally:
+            return msg
```

Comment:
remove the `<red>` tags, and change the `<orange>` tags to be `<highlight>` tags